apps/testing-javascript/schemas/objects/castingwordsTranscript.js

```diff
@@ -20,7 +20,7 @@ export default {
     {
       name: 'transcript',
       title: 'Transcript',
-      type: 'body',
+      type: 'text',
```

Comment:
Thanks for pointing this out. It makes sense that we are moving away from the `castingwords` object. I'm leaning toward leaving TJS as is to avoid having to migrate the data in Sanity and update a few GROQ queries. If you have a case for why it should be migrated for TJS, I'm open to hearing that.

> In this case it could just be matter of renaming it?

I'm pretty sure I'd have to write and run a `sanity exec` migration to move the `transcript` and `srt` values for all VideoResources to be under `transcript` instead of under `castingwords`.